superset-frontend/src/dashboard/components/SaveModal.tsx

```diff
@@ -185,7 +207,7 @@ class SaveModal extends React.PureComponent {
             <div className="m-l-25 m-t-5">
               <Checkbox
                 checked={this.state.duplicateSlices}
-                onChange={this.toggleDuplicateSlices}
+                onChange={() => this.toggleDuplicateSlices}
```

Comment:
wondering why this was needed, seems like it might break it (not that toggleDuplicateSlices isn't actually being called I think)